.github/ISSUE_TEMPLATE/3_Support_question.md

```diff
@@ -0,0 +1,10 @@
+---
+name: ⛔ Support Question
+about: See [PX4 Discuss](http://discuss.px4.io/) or [Slack](http://slack.px4.io/) for questions about using PX4.
+
+---
+
+We use GitHub issues only to discuss PX4 bugs and new features. For
+questions about using PX4 or related components, please use [PX4 Discuss](http://discuss.px4.io/) or [Slack](http://slack.px4.io/).
```

Comment:
Should we even mention slack in the first instance? I've been trying to push people more to discuss. Up to you!

Reply:
Let's start there and see how it goes.